brainiak/reprsimil/brsa.py

```diff
@@ -62,22 +198,26 @@ class BRSA(BaseEstimator):
     will be provided as a quantification of neural representational similarity.
 
     .. math::
-        Y = X \\cdot \\beta + \\epsilon
+        Y = X \\cdot \\beta + X_0 \\cdot \\beta_0 + \\epsilon
 
         \\beta_i \\sim N(0,(s_{i} \\sigma_{i})^2 U)
 
     Parameters
     ----------
-    n_iter : int, default: 200
+    n_iter : int. Default: 50
         Number of maximum iterations to run the algorithm.
-    rank : int, default: None
+    n_iter_inner : int, default: 6
+        Number of maximum iterations in the inner loop of optimization (within
+        each iteration of n_iter). Users typically do not need to adjust this
+        parameter.
+    rank : int. Default: None
         The rank of the covariance matrix.
```

Comment:
It is a degenerate likelihood. I did some reparametrization: beta = (s*sigma) * L * alpha. alpha ~ N(0, I). To make the covariance matrix U low rank, we basically restrict the number of columns of L to be the rank chosen by the user. Then alpha is also lower dimensional.